server/database/management/declarations.js

```diff
@@ -1,6 +1,11 @@
+// autogenerated by sql-generate v1.1.0 on Wed Dec 09 2015 16:38:42 GMT+0000 (UTC)
```

Comment:
Comments have been re-enabled for this file when it's generated for this line specifically.
We can imagine in the future adding a pre-commit hook that verifies that **any** change to `createTables.sql` has a corresponding change to `declarations.js`. 
In some cases changes to `createTables.sql` do not need a change to declarations (cosmetic changes, indices, adding constraints, etc.). Changes to this comment line will make the pre-commit hook happy.